src/equilibrate_utils.jl

```diff
@@ -0,0 +1,726 @@
+
+## args
+-  solver (optional): a subtype of AbstractBifurcationSolver that specifies how to solve the bifurcation diagram. Default is HCBifurcationSolver.
+-  reaction_network: a reaction network.
+-  parameter_values: a vector which specifies the point in parameter space around which the bifurcation diagram is evaluated.
+-  parameter_symbol: the parameter which is varied, given as a symbol.
+-  parameter_range: the range over which the specified parameter is varied.
+
+## kwargs
+-  dp=(parameter_range[2] - parameter_range[1])/200: The distance to jump after finding a bifurcation. After discovering a bifurcation, the solver jumps ahead a distance `dp` and starts back-tracking. If this distance is too small, the method may error or cause visible artifiacts in the bifurcation diagram. If it is too large, then you might jump over another bifurcation and it will be missed (this should also be farily obvious in a plot). Only applicable for the HCBifurcationSolver.
+- d_sol = 1e-7: (alternatively reduced to a tenth of the minimum value of any initial solutions) used by algorithm to determine if two points are identical. Set this to less than the expected minimum distance between two different solutions.
+"""
+function bifurcations(solver::AbstractBifurcationSolver, rn::DiffEqBase.AbstractReactionNetwork, p, param::Symbol, range; kwargs...)
+    return BifurcationDiagram(param,range,solve_bifurcation(solver,rn,p,param,range; kwargs...))
+end
+
+function bifurcations(rn::DiffEqBase.AbstractReactionNetwork, args...; kwargs...)
+    return bifurcations(HCBifurcationSolver(), rn, args...; kwargs...)
+end
+
+#Generates a grid of bifurcation points, using a given steady state method.
+"""
+    bifurcation_grid([solver], reaction_network, parameter_values, parameter_symbol, parameter_range; kwargs...)
+
+    Solves the steady states of a system for every parameter values in a given range of values. Returns a bifurcation grid type object (which can be plotted).
+
+## args
+ - solver (optional): a subtype of AbstractSteadyStateSolver that specifies how to solve the bifurcation diagram. Default is HCSteadyStateSolver (currently only solver avaiable).
+ - reaction_network: a reaction network.
+ - parameter_values: a vector which specifies the parameter values for which the steady states are to be found. Not required in case the reaction network lacks steady states.
+ - parameter_symbol: the parameter which is to be varried.
+ - parameter_range: a range with all the values of the parameter for which steady states are to be solved.
+
+## kwargs
+ - potential arguments for steady state solvers (however, only current solver does not have any such arguments).
+"""
+function bifurcations_grid(rn::DiffEqBase.AbstractReactionNetwork, args...)
+    return bifurcations_grid(HCSteadyStateSolver(), rn, args...)
+end
+function bifurcations_grid(solver::AbstractSteadyStateSolver, rn::DiffEqBase.AbstractReactionNetwork,p::Vector{Float64},param::Symbol,range::AbstractRange)
```

Comment:
network then solver to match conventions?

Reply:
Yes, I have shifted this.